internal/output/html/package_table_template.html

```diff
@@ -1,53 +1,62 @@
 <table class="vuln-table">
   <tr>
+    <th></th>
     <th>Package</th>
-    <th>Ecosystem</th>
     <th>Installed version</th>
-    <th>Fixed version</th>
+    <th>Fixed available</th>
     <th>Vulnerability Count</th>
   </tr>
   {{ range $rowIndex, $element := . }}
-  {{ if eq (len $element.CalledVulns) 0 }}
-  {{ continue }}
-  {{ end }}
-
-  <tr class='vuln-tr' id="vuln-tr-{{ $element.Name}}-{{ $rowIndex }}"
-    onclick="showVulnDetails('{{ $element.Name }}-{{ $rowIndex }}')">
-    <td>{{ $element.Name }}</td>
-    <td>{{ $element.Ecosystem }}</td>
-    <td>{{ $element.InstalledVersion }}</td>
-    {{ if ne $element.HTMLVulnCount.UnFixed 0 }}
-    {{ if eq $element.HTMLVulnCount.Fixed 0}}
-    <td class="warning">
-      {{ $element.FixedVersion }}
+  {{ $index := uniqueID }}
+  <tr class="table-tr package-tr {{if $element.HTMLPackageDetail }}has-layer-info"
+    data-layer="{{ $element.HTMLPackageDetail.LayerID }}{{ end }}" id="table-tr-{{ $index }}">
+    <td class="icon-td">
+      <div class="expand-icon" onclick="showPackageDetails('{{ $index }}')">
```

Comment:
Would it make sense to have the details expand if we click anywhere in this entire row? 

Right now, we can only expand if we click on the rather small triangle, which also feels like it's not ideal for accessibility.

Reply:
Updated